client/my-sites/plugins/plugin-management-v2/plugin-row-formatter/style.scss

```diff
@@ -86,9 +86,6 @@ a.button.plugin-row-formatter__plugin-name-card {
 	}
 
 	.components-form-toggle {
-		position: absolute;
-		inset-inline-start: 200px;
-
 		@include break-xlarge {
 			position: relative;
 			inset-inline-start: unset;
```

Comment:
In that case, these can be removed too.

Reply:
True, removing it

Reply:
Done as part of 3e139b7bbfbbad157be6bfb35d6b6c2bd233f528